sanitychecks.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2016 Tuenti Technologies S.L. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+        "log"
+        "strings"
+        "strconv"
+
+	"k8s.io/client-go/pkg/api/v1"
+        "github.com/achanda/go-sysctl"
+)
+
+var (
+        ephemeralPortsRange *EphemeralPortsRange
+)
+
+type EphemeralPortsRange struct {
+        check    bool
+        LowPort  int32
+        HighPort int32
+}
+
+// Retrieve the data from sysctl and return the values, disable the check if unseccseful and log
+func initEphemeralPortsRangeCheck() *EphemeralPortsRange {
+        r, err := sysctl.Get("net.ipv4.ip_local_port_range")
```

Comment:
I know (in fact it's what the package is doing)... I'll keep it if you don't mind.